README.md

```diff
@@ -10,22 +10,19 @@
 <!--END STABILITY BANNER-->
 
 This library provides a high level, extensible pattern for constructing services
-deployed using Amazon ECS.
+deployed using Amazon ECS. It is named `@aws-cdk-containers/ecs-service-extensions` for
+TypeScript and JavaScript and named `cdk-ecs-service-extensions` for other JSII target
+languages.
```

Comment:
I am concerned with the automatic translation of the import statements in the readme on construct hub. This will be incorrect for non node languages and I feel like it's worth calling out.

Reply:
i think i'm wrong about this actually. I'll remove the confusing part.